3/debian-10/rootfs/opt/bitnami/scripts/libmoodle.sh

```diff
@@ -184,7 +204,7 @@ EOF
         db_name="$(moodle_conf_get "\$CFG->dbname")"
         db_user="$(moodle_conf_get "\$CFG->dbuser")"
         db_pass="$(moodle_conf_get "\$CFG->dbpass")"
-        [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" ]] && moodle_wait_for_mysql_db_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
+       [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" || "$db_type" = "pgsql" ]] && moodle_wait_for_db_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
```

Comment:
I would recommend to create a new function for checking PostgreSQL connections, i.e.:

```suggestion
        [[ "$db_type" = "mariadb" || "$db_type" = "mysqli" ]] && moodle_wait_for_mysql_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
        [[ "$db_type" = "pgsql" ]] && moodle_wait_for_postgresql_connection "$db_host" "$db_port" "$db_name" "$db_user" "$db_pass"
```